ga4gh/exceptions.py

```diff
@@ -447,7 +447,18 @@ class MissingReferenceMetadata(MalformedException):
     """
     def __init__(self, fileName, key):
         self.message = (
-            "JSON metadata for file {} is missing key {}".format(
+            "JSON reference metadata for file {} is missing key {}".format(
+                fileName, key))
+
+
+class MissingReferenceSetMetadata(MalformedException):
+    """
+    A FASTA file is missing some metadata in the corresponding JSON file.
```

Comment:
This comment is a bit misleading --- this is a directory containing FASTAs, not a single fasta file.